dashboard/app/controllers/learning_goal_teacher_evaluations_controller.rb

```
@@ -0,0 +1,47 @@
+class LearningGoalTeacherEvaluationsController < ApplicationController
+  load_and_authorize_resource
+
+  def create
+    @learning_goal_teacher_evaluation.update(learning_goal_teacher_evaluation_params.merge(teacher_id: current_user.id))
```

do you want `update!` here? I'd be worried a failed update could be silently dropped otherwise. Or, maybe since the resulting, non-updated object would be sent to the client, everything would be ok?

---

yes thank you!